<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyFindForm</class>
 <widget class="QWidget" name="MyFindForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLineEdit" name="le_address"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_find_programm">
       <property name="text">
        <string>find</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="le_screenshot"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn_set_screenshot_name">
       <property name="text">
        <string>shot</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="pixmap">
          <pixmap resource="../../../../targets/targets.qrc">:/targets/ok.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="pixmap">
          <pixmap resource="../../../../targets/targets.qrc">:/targets/auto.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="pixmap">
          <pixmap resource="../../../../targets/targets.qrc">:/targets/в бой.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_find_auto">
         <property name="text">
          <string>find</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn_find_ok">
         <property name="text">
          <string>find</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btn_find_to_battle">
         <property name="text">
          <string>find</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>244</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_address</tabstop>
  <tabstop>btn_find_programm</tabstop>
  <tabstop>le_screenshot</tabstop>
  <tabstop>btn_set_screenshot_name</tabstop>
  <tabstop>btn_find_auto</tabstop>
  <tabstop>btn_find_ok</tabstop>
  <tabstop>btn_find_to_battle</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../targets/targets.qrc"/>
 </resources>
 <connections/>
</ui>
